runtime/parser2/type.go

```diff
@@ -141,6 +141,14 @@ func defineType(def interface{}) {
 }
 
 func init() {
+	var i lexer.TokenType
+	for i = 0; i < lexer.TokenMax; i++ {
+		typeLeftBindingPowers[i] = 0
+		typeNullDenotations[i] = nil
+		typeLeftDenotations[i] = nil
+		typeMetaLeftDenotations[i] = nil
+	}
+
```

Comment:
This initialization is not needed, Go initializes the arrays to the default value automatically

```suggestion
```